Cut 0.0.99

TheBlueMatt opened the pull request:
This assumes #975 lands first.

ariard commented on CHANGELOG.md:
```diff
@@ -1,12 +1,69 @@
    `ChannelManager` will fail to deserialize both on version 0.0.98 and later
    versions. If you have such a `ChannelManager` available, a simple patch will
-   allow it to deserialize, please file an issue if you need assistance.
+   allow it to deserialize, please file an issue if you need assistance (#973).
```
Should you mention edbc30c which is bumping chan's serialization ?

TheBlueMatt replied:
Nope, its fully backwards- and forwards-compatible. 0.0.98 can read versions written by 0.0.99 (though it ignores the fee info) and 0.0.99 can read versions written by 0.0.98.

jkczyz commented on CHANGELOG.md:
```diff
@@ -1,12 +1,69 @@
+## Node Compatibility
+
+ * We now accept spurious `funding_locked` messages sent prior to
+   `channel_reestablish` messages after reconnect. This is a [known,
+   long-standing bug in lnd]
+   (https://github.com/lightningnetwork/lnd/issues/4006) (#966).
```
Link isn't being rendered, possibly because of the whitespace after `]`.

jkczyz commented on CHANGELOG.md:
```diff
@@ -1,12 +1,69 @@
+
+ * We now accept spurious `funding_locked` messages sent prior to
+   `channel_reestablish` messages after reconnect. This is a
+   [known, long-standing bug in lnd] (https://github.com/lightningnetwork/lnd/issues/4006)
```
I think you need to remove the space between the `]` and the `(`, otherwise the text is not linked:

https://github.com/rust-bitcoin/rust-lightning/blob/946ca58a59aed15169b22208fa7dc58f7cfc1cc2/CHANGELOG.md

jkczyz commented on CHANGELOG.md:
```diff
@@ -1,12 +1,69 @@
+## API Updates
+
+ * `lightning_block_sync::poll::Validate` is now public, allowing you to
+   implement the `lightning_block_sync::poll::Poll` trait yourself (#956).
```
s/yourself/without `lightning_block_sync::poll::ChainPoller`

jkczyz commented on CHANGELOG.md:
```diff
@@ -1,12 +1,69 @@
    `ChannelManager` will fail to deserialize both on version 0.0.98 and later
    versions. If you have such a `ChannelManager` available, a simple patch will
-   allow it to deserialize, please file an issue if you need assistance.
+   allow it to deserialize, please file an issue if you need assistance (#973).
```
It doesn't look like adding PR/issue numbers adds links

TheBlueMatt replied:
Hmm, is it necessary? Its kinda annoying to have a million links in a plaintext document, and its not hard to load the PR from the number.

jkczyz replied:
Probably not necessary but I'd think it's less annoying since it is essentially write once, read many times.

jkczyz commented on CHANGELOG.md:
```diff
@@ -1,12 +1,70 @@
 # 0.0.99 - WIP
```
Add date?

jkczyz commented on CHANGELOG.md:
```diff
@@ -1,12 +1,73 @@
+   channels, the value is set to the value provided in
+   `ChannelManagerReadArgs::default_config::channel_options` the first time the
+   `ChannelManager` is loaded in 0.0.99 (#975).
+ * We now reject HTLCs which are received to be forwarded out over private
```
nit: s/out over/over

jkczyz commented on CHANGELOG.md:
```diff
@@ -1,12 +1,73 @@
    `ChannelManager` will fail to deserialize both on version 0.0.98 and later
    versions. If you have such a `ChannelManager` available, a simple patch will
-   allow it to deserialize, please file an issue if you need assistance.
+   allow it to deserialize, please file an issue if you need assistance (#973).
```
nit: s/, please/. Please

jkczyz reviewed (approved):
Just some nits.

valentinewallace reviewed (approved):
ACK mod jeff's nits

TheBlueMatt commented:
Fixed nits and squashed. Will take after CI:
```
$ git diff-tree -U1 83110e35 c9a8b26d
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 0fa93a84..764e2ffd 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,2 +1,2 @@
-# 0.0.99 - WIP
+# 0.0.99 - 2021-07-09
 
@@ -25,5 +25,5 @@
    `ChannelManager` is loaded in 0.0.99 (#975).
- * We now reject HTLCs which are received to be forwarded out over private
-   channels unless `UserConfig::accept_forwards_to_priv_channels` is set. Note
-   that `UserConfig` is never serialized and must be provided via
+ * We now reject HTLCs which are received to be forwarded over private channels
+   unless `UserConfig::accept_forwards_to_priv_channels` is set. Note that
+   `UserConfig` is never serialized and must be provided via
    `ChannelManagerReadArgs::default_config` at each start (#975).
@@ -69,3 +69,3 @@
    versions. If you have such a `ChannelManager` available, a simple patch will
-   allow it to deserialize, please file an issue if you need assistance (#973).
+   allow it to deserialize. Please file an issue if you need assistance (#973).
 
```
